### DISCUSSION TITLE
建议 增加一个访问密码

### DISCUSSION THREAD

**Discussion by dianso** [gaowei-space/markdown-blog#35] [Ideas]
这几天设置好了光猫桥接，默认分配240开始的ipv6

我在外也能访问家里nas上的markdown-blog了

这几天看百度统计，好多IP访问，不知道是机器人还是什么的扫到了端口，总感觉不太安全

能不能加一个访客密码或者给某个文件夹增加一个密码功能

**Comment by gaowei-space** [comment]
如果作为个人博客和文档，有浏览量是好事啊。我的建议是隐私内容建议就不要放到博客上了，说实在的现在的网络环境没什么是绝对隐私的了 😅

**Reply by dianso** [reply]
> 如果作为个人博客和文档，有浏览量是好事啊。我的建议是隐私内容建议就不要放到博客上了，说实在的现在的网络环境没什么是绝对隐私的了 😅

好吧
